packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -33,24 +34,93 @@ Requires:       iptables
+Provides:       bundled(golang(github.com/aws/aws-sdk-go/service/sts))
+Provides:       bundled(golang(github.com/Azure/go-ansiterm))
+Provides:       bundled(golang(github.com/Azure/go-ansiterm/winterm))
+Provides:       bundled(golang(github.com/cihub/seelog))
+Provides:       bundled(golang(github.com/cihub/seelog/archive))
+Provides:       bundled(golang(github.com/cihub/seelog/archive/gzip))
+Provides:       bundled(golang(github.com/cihub/seelog/archive/tar))
+Provides:       bundled(golang(github.com/cihub/seelog/archive/zip))
+Provides:       bundled(golang(github.com/docker/docker/api/types))
+Provides:       bundled(golang(github.com/docker/docker/api/types/blkiodev))
+Provides:       bundled(golang(github.com/docker/docker/api/types/container))
+Provides:       bundled(golang(github.com/docker/docker/api/types/filters))
+Provides:       bundled(golang(github.com/docker/docker/api/types/mount))
+Provides:       bundled(golang(github.com/docker/docker/api/types/network))
+Provides:       bundled(golang(github.com/docker/docker/api/types/registry))
+Provides:       bundled(golang(github.com/docker/docker/api/types/strslice))
+Provides:       bundled(golang(github.com/docker/docker/api/types/swarm))
+Provides:       bundled(golang(github.com/docker/docker/api/types/versions))
+Provides:       bundled(golang(github.com/docker/docker/opts))
 Provides:       bundled(golang(github.com/docker/docker/pkg/archive))
 Provides:       bundled(golang(github.com/docker/docker/pkg/fileutils))
+Provides:       bundled(golang(github.com/docker/docker/pkg/homedir))
+Provides:       bundled(golang(github.com/docker/docker/pkg/idtools))
 Provides:       bundled(golang(github.com/docker/docker/pkg/ioutils))
+Provides:       bundled(golang(github.com/docker/docker/pkg/jsonlog))
+Provides:       bundled(golang(github.com/docker/docker/pkg/jsonmessage))
+Provides:       bundled(golang(github.com/docker/docker/pkg/longpath))
+Provides:       bundled(golang(github.com/docker/docker/pkg/mount))
 Provides:       bundled(golang(github.com/docker/docker/pkg/pools))
 Provides:       bundled(golang(github.com/docker/docker/pkg/promise))
+Provides:       bundled(golang(github.com/docker/docker/pkg/stdcopy))
 Provides:       bundled(golang(github.com/docker/docker/pkg/system))
-Provides:       bundled(golang(github.com/docker/docker/pkg/units))
-Provides:       bundled(golang(github.com/docker/docker/vendor/src/code.google.com/p/go/src/pkg/archive/tar))
+Provides:       bundled(golang(github.com/docker/docker/pkg/term))
+Provides:       bundled(golang(github.com/docker/docker/pkg/term/windows))
+Provides:       bundled(golang(github.com/docker/go-connections/nat))
+Provides:       bundled(golang(github.com/docker/go-units))
 Provides:       bundled(golang(github.com/fsouza/go-dockerclient))
+Provides:       bundled(golang(github.com/go-ini/ini))
 Provides:       bundled(golang(github.com/golang/mock/gomock))
-Provides:       bundled(golang-logrus)
```

Comment:
If I understand the packaging recommendations surrounding the use of bundled and de-bundled libraries, this would be `bundled(golang-logrus)` only if our package was consuming the named rpm for compilation, but we use our own vendored bundled copy.